LibreNMS/Alert/Transport/Ciscospark.php

```diff
@@ -12,18 +12,20 @@
 namespace LibreNMS\Alert\Transport;
 
 use LibreNMS\Interfaces\Alert\Transport;
+use LibreNMS\Alert\AlertUtil;
 
 class Ciscospark implements Transport
 {
     public function deliverAlert($obj, $opts)
     {
-        $token  = $opts['token'];
-        $roomId = $opts['roomid'];
-        $text   = strip_tags($obj['msg']);
-        $data   = array(
-            'roomId' => $roomId,
+        $details = AlertUtil::getTransportConfig($opts['alert']['transport_id']);
```

Comment:
We should try and move this into the Transport class, maybe in __construct() otherwise we're just duplicating the code all the time.